tests/lib/rules/no-extra-bind.js

```diff
@@ -25,7 +25,10 @@ ruleTester.run("no-extra-bind", rule, {
         "var a = function() { this.b }.foo()",
         "var a = f.bind(a)",
         "var a = function() { return this.b }.bind(c)",
-        { code: "var a = (() => { return b }).bind(c, d)", parserOptions: { ecmaVersion: 6 } }
+        { code: "var a = (() => { return b }).bind(c, d)", parserOptions: { ecmaVersion: 6 } },
+        "(function() { (function() { this.b }.bind(this)) }.bind(c))",
+        "var a = function() { return 1; }[bind](b)",
```

Comment:
Thank you for reviewing!

This test verifies this rule ignores computed properties.

```js
var bind = "foo";
var a = function() { return 1; }[bind](b); // this shouldn't be warned!
```

The function does not have `this` keywords, if it's `"bind"` then this should be warned.
....... but not warned currently.
I will fix it.